extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -412,5 +418,10 @@ describe('amp-analytics.visibility', () => {
       expect(getElement('div', analytics, 'scope')).to.equal(null);
       expect(getElement('amp-img', analytics, 'scope')).to.equal(img2);
     });
+
+    it.skip('finds element for selectionMethod=host', () => {
```

Comment:
To initialize iframe fully what you need to do is this:
```
beforeEach(() => {
  const loaded = loadPromise(iframe);
  container.appendChild(iframe);
  return loaded;
});
```

See an example here: https://github.com/ampproject/amphtml/blob/master/test/functional/test-extensions.js#L398